sdk/docs/manually-written/overview/overview/index.rst

```diff
@@ -3,30 +3,105 @@
+* **Regulatory Compliance**: The data is shared, stored and operated by the parties owning it. This allows to comply with existing regulations of the jurisdictions where the parties operate.
+
+Continue diving into the :externalref:`Canton requirements <requirements>` and learn more about how Canton
+is :brokenref:`different from other ledgers <comparison-blockchain>`.
+
+Elements of the Canton Network
+------------------------------
+
+The **Canton Network** is the union of all applications and users that are built and operated using the software **Canton**,
+which connects individual instances through the **Canton Protocol**.
+
+The shared data items managed and synchronized by Canton are called **contract instances**. Every contract instance is
+associated with a **template** that defines the **smart contract choices** for the contract instance. These choices
+represent the possible update commands on the ledger, archiving a given set of contracts and creating new ones
+atomically. These rules are written in the **DAML** programming language and contain the business logic and authorization
+rules.
+
+The entirety of the contract instances and templates is called the **Ledger**. Each contract is owned by a group of
+stakeholders, called **Parties**. Each party participates in the network through their validator node. This validator
+node has only visibility to the subset of contracts of the parties associated. As validator nodes are able to connect
+to different networks, the entire network forms a single **Virtual Global Ledger** which is not controlled by any
+single entity.
+
+The software **Canton** has two infrastructure components: **Synchronizers** and **Validators** (formerly named participants).
+Validators process and validate transactions, managing the ledger state of their parties. Validators do not expose any public port,
+but instead connect to synchronizers to form a network with other validators. Each validator can connect to multiple
+synchronizers unilaterally, creating a network of networks.
+
+In order to transact with each other on a given transaction, validators need to find at least one synchronizer where
+they are all connected to. This synchronizer will then order, **buffer and forward the encrypted messages** to the respective
+validators without understanding what is being transacted on. There is one specific instance of a synchronizer that is
+called the **Global Synchronizer** (`https://sync.global <https://sync.global>`_), serving as the global backbone of the network.
+
+Contracts are not tied to synchronizers. Synchronizers are not aware of each other. Instead, validators can use any
+suitable synchronizer for their transactions, creating an **natively interoperable network** of synchronizers and validators.
+This allows the Canton Network to **scale horizontally** by just adding more synchronizers and validators whenever needed.
+
```

Comment:
Consider to add another `Continue diving ...` with a link to https://github.com/DACH-NY/canton/blob/release-line-3.3/docs-open/src/sphinx/overview/overview/canton.rst. WDYT?